src/components/Guest/Guest.tsx

```diff
@@ -14,14 +15,15 @@ const Guest: React.FC<Props> = ({
 }) => {
   return (
     <article className="guestCard">
-      <h1>{guestName}</h1>
-      <h2>{phoneNumber}</h2>
+      <h1 className="guestInfo">{guestName}</h1>
+      <h2 className="guestInfo">{phoneNumber}</h2>
```

Comment:
I don't think will cause a merge conflict with my branch, but if it does, it's an easy fix